Trell.Engine/Extensibility/Interfaces/IStorageProvider.cs

```diff
@@ -6,12 +6,12 @@ namespace Trell.Engine.Extensibility.Interfaces;
 public interface IStorageProvider {
     int MaxDatabasePageCount { get; }
 
-    bool TryResolvePath(
+    bool TryResolveTrellPath(
```

Comment:
Why the name change? `TrellPath` is a different class that isn't referenced in this signature.

Reply:
TrellPath is a specific type of path that is intentionally very limited, so including that aspect in the name here should help set expectations about what you should pass in. And the type does come up in the filesystem implementation.

Reply:
Yeah, alright. That makes sense to me.